target_chains/solana/sdk/js/pyth_solana_receiver/README.md

```diff
@@ -0,0 +1,61 @@
+# Pyth Solana Receiver JS SDK
+
+This is a Javascript SDK to interact with the Pyth Solana Receiver contract on Solana whose code lives [here](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/solana).
+
+## Pull model
+
+The Pyth Solana Receiver allows users to consume Pyth price updates on a pull-basis. This means that the user is responsible for pushing the price data on-chain whenever they want to interact with an app that requires a price update.
```

Comment:
pull basis (no hyphen)

and i think the user should be responsible for "pulling" the price data on chain? or maybe "submitting"?